### PR TITLE
Fixed installation issue in AArch64

### PR DESCRIPTION
**Pull request by quic-amitraj**
Added explicit packages with suitable versions.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -33,6 +33,8 @@ dependencies = [
     "protobuf==3.20.2",
     "onnxscript==0.1.0.dev20240327",
     "sympy",
+    "networkx==3.1",
```
Comment (vbaddi) [inline pyproject.toml:36]: These are not required, will be downloaded as part of torch.
also `torch==2.0.0` is not allowed, we need to explicitly pass `+cpu`

**Review by vbaddi — CHANGES_REQUESTED** [review]

File: pyproject.toml (hunk 1 above)
Comment (quic-amitraj) [reply pyproject.toml:36]: Yes, it should but throwing error-
image

Comment (vbaddi) [reply pyproject.toml:36]: Pls check the readme, these changes are not required, pls revert the PR.

File: pyproject.toml [hunk 2]
```diff
@@ -33,6 +33,8 @@ dependencies = [
...
+    "multiprocess==0.70.14",
```
Comment (ochougul) [inline pyproject.toml:37]: add dill package, otherwise throws error on aarch64 platform

**Comment by ochougul** [comment]
Waiting for Carls reply

**Pull request closed without merging**